lib/node_modules/@stdlib/stats/incr/nancovariance/lib/main.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable stdlib/jsdoc-no-space-aligned-asterisks */
```

Comment:
Do NOT disable this lint rule. The lint rule is telling you explicitly our conventions.